.gitignore

```diff
@@ -17,16 +17,9 @@ vulcan-tracker
 
 # Dependency directories (remove the comment below to include it)
 # vendor/
-.vscode/
 out/
 
-
 dist/
-.env
-.DS_Store
-.history
-.idea
-
 
```

Comment:
Why do you remove this? i.e. I have a .vscode directory that I want to ignore.
I would keep all of them.

Reply:
It's better to have a global configuration in the git installation on your computer than to move the specifics of each development environment into the repository. More info: https://stackoverflow.com/questions/7335420/global-git-ignore

Reply:
Ok, makes sense, I ll have to setup my env !